cmake/shared/PACE_AddMex.cmake

```diff
@@ -85,15 +78,11 @@ function(pace_add_mex)
         DOCUMENTATION "${${prefix}_DOCUMENTATION}"
         LINK_TO "${${prefix}_LINK_TO}"
     )
-
-    if(${prefix}_COPY_TO)
-        set(_target_file "$<TARGET_FILE:${${prefix}_NAME}>")
-        set(_dest_file "${${prefix}_COPY_TO}/$<TARGET_FILE_NAME:${${prefix}_NAME}>")
-        add_custom_command(TARGET "${${prefix}_NAME}"
-            POST_BUILD
-            COMMAND ${CMAKE_COMMAND} -E copy "${_target_file}" "${_dest_file}"
-            COMMENT "Copying ${${prefix}_NAME}.${Matlab_MEX_EXTENSION} into ${${prefix}_COPY_TO}"
+    set_target_properties("${${prefix}_NAME}"
+        PROPERTIES
+            # Appending the $<0:> generator expression prevents MSVC outputing
```

Comment:
Rather than saying what it prevents, this comment should say what it causes to happen, i.e. build artifacts are placed in the XX_OUTPUT_DIRECTORY folder rather than the normal Release or Debug folder